examples/complete/versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.40"
+      version = ">= 5.0"
```

Comment:
Why is this change necessary? I don't think you introduce anything backward incompatible with v4.

Reply:
You're right, it's not necessary, sorry. I've reverted this v5 upgrade.